### ISSUE TITLE
Serving main.js gets very slow (with Bootstrap?)

### ISSUE THREAD

**Issue by miro** [jakemmarsh/angularjs-gulp-browserify-boilerplate#44]
While developing an app based on this boilerplate, I'm experiencing frustratingly long load times with the `gulp serve`.

I've added [Bootflat](http://bootflat.github.io/) to my application, I'm not sure yet if this has something to do with it.

After I've started gulp with `gulp serve` -command, and then change some lines, gulp output is like this:

     [BS] Reloading Browsers...
     [BS] 0 file changed ()
     [BS] Reloading Browsers...
     [BS] 0 file changed ()
     [BS] Reloading Browsers...
     [BS] 0 file changed ()
     [BS] Reloading Browsers...
     [BS] 0 file changed ()
     GET /bower_components/bootflat/css/bootflat.css 200 4.313 ms - 37162
     GET /css/main.css 200 4.440 ms - 3822
     GET /bower_components/bootflat/css/font-awesome.css 200 5.197 ms - 22694
     GET /bower_components/bootflat/bootstrap/bootstrap.css 200 4.992 ms - 121220
     GET /bower_components/bootflat/bootstrap/bootstrap.css.map 304 3.235 ms - -
     GET /js/main.js 200 95.821 ms - 5228067
     GET /assets/MetricWeb-Light.woff 200 0.927 ms - 44190

Notice the over one and half a minute wait before the main.js is server!

After that when refreshing the page on the browser, those same resources get served ~instantly.

Any ideas about this one? Am I doing some utterly wrong?

**Comment by jakemmarsh** [comment]
I'm not familiar with Bootflat in general, but adding a CSS library shouldn't be causing this.

What is the `gulp serve` command? You should be running `gulp dev` while developing.

**Comment by miro** [comment]
Sorry, I got confused with another gulp setup I'm using, ignore the gulp command, I am using the `gulp dev` also.

The main.js indeed seems to get very big, ~5.0M, I have to do some research what is getting in there. 

(Should have looked there before submitting this, I thought that the reason could be the usage of Bootflat, since the slowness started when I added it to the project).

I'm closing this for now, I'll get back at you if this appears to be a problem with the boilerplate itself. Thanks for your quick reply.
